variants/t-eth-elite/platformio.ini

```diff
@@ -0,0 +1,16 @@
+[env:t-eth-elite]
+extends = esp32s3_base
+board = esp32s3box
+board_check = true
+build_flags = 
+  ${esp32s3_base.build_flags}
+  -D PRIVATE_HW
```

Comment:
Private for now?

Reply:
Wanted to wait for review :-) - Also the GPS still gives me autodetct headache. the BME280 is found fine.